track_local_static.go

```diff
@@ -128,12 +128,14 @@ func (s *TrackLocalStaticRTP) WriteRTP(p *rtp.Packet) error {
 
 // writeRTP is like WriteRTP, except that it may modify the packet p
 func (s *TrackLocalStaticRTP) writeRTP(p *rtp.Packet) error {
+	bindings := make([]trackBinding, len(s.bindings))
```

Comment:
```
bindings := append([]trackBinding(nil), s.bindings...)
```

It is unfortunate we need to copy :( what is the deadlock? Curious if we could fix another way